commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/ShoppingListDraft.java

```diff
@@ -0,0 +1,40 @@
+package io.sphere.sdk.shoppinglists;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.FactoryMethod;
+import io.sphere.sdk.annotations.ResourceDraftValue;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.types.CustomFieldsDraft;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+@JsonDeserialize(as = ShoppingListDraftDsl.class)
+@ResourceDraftValue(factoryMethods = {
+        @FactoryMethod(parameterNames = {"name"}),
+})
+public interface ShoppingListDraft {
+
+    LocalizedString getName();
+
+    @Nullable
+    LocalizedString getDescription();
+
+    @Nullable
+    String getKey();
+
+    @Nullable
+    Reference<Customer> getCustomer();
+
+    @Nullable
+    LocalizedString getSlug();
+
+    List<LineItemDraft> getLineItems();
```

Comment:
I think it's possible to omit the field completely, and it should be possible to send `lineItems: null`. Please tell me if it is not the case.
We change the JSON parser a few months ago when Michael found out that the behaviour was different if the field was omitted or if the field was set the `null`. Now it should behave the same in both cases.